src/components/fileUpload.js

```diff
@@ -229,7 +394,14 @@
 
     B.defineFunction('clearFileUpload', e => clearFiles(e));
 
-    return isDev ? <div className={classes.root}>{Control}</div> : Control;
+    return isDev ? (
+      <div>
+        <div className={classes.root}>{Control}</div>
+        <DevUploadedFile />
+      </div>
+    ) : (
+      Control
```

Comment:
This looks a bit weird to me. I think it should be: `<Control />`?

Reply:
In order to use `<Control />` in that way, I've changed the way that `Control` was defined, from `const Control = (...)` to `const Control = () => (...)`.